lib/ketting.js

```diff
@@ -200,7 +167,7 @@ Ketting.prototype = {
         request.headers.set('Authorization', 'Bearer ' + this.auth.token);
         break;
       case 'oauth2' :
-        return oauth.fetch(this, request, init);
```

Comment:
Why is the `init` being removed from the oauth calls?

Reply:
The reason is that `init` is really an override of the settings already being padded in the `Request` object. Basically `Request` is a template of the actually request being made, and `init` is a list of settings that `win`. There's several points earlier in the code that make sure that `init` is honored.